### DISCUSSION TITLE
PubSubHubBub v3

### DISCUSSION THREAD

**Discussion by JoannSellerServices** [etsy/open-api#695] [Q&A]
With our migration to v3, I don't see any reference to the Etsy Superfeedr PubSubHubBub in retired or active resources.  There doesn't seem to be any reference to it at all.  I'm wondering, if our application is utilizing this for real-time feed updates, will this be available in v3?  If not, is there a suggested workaround to emulate this resource?  Any direction is greatly appreciated!

**Comment by benjymouse** [comment]
We've been using this for a little while now and find it super useful.

**Comment by sampok** [comment]
We are also relying on the v2 real-time feed. Instead of PubSubHubbub we're using [the /listings/latest endpoint](https://www.etsy.com/developers/documentation/getting_started/real_time_feeds).

Waiting for news on [v3 real-time feed](https://github.com/etsy/open-api/issues/75) or some type of a [workaround solution](https://github.com/etsy/open-api/issues/528).

**Comment by etsyachristensen** [comment]
We'll close this discussion out as it's already being discussed here: https://github.com/etsy/open-api/discussions/679
